### PR TITLE
Add delete/resume/view to experiment-management

### PR DESCRIPTION
**Pull request by Why1005**
### Checklist ###
  - [ ] test case
  - [ ] doc

### CODE REVIEW

File: ts/webui/src/static/style/create/topTrial.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+    .circle {
+        width: $circle;
+        height: $circle;
+        border-radius: 50%;
+        background-color: $bgblue;
+        margin-top: 6px;
+        margin-right: 18px;
+    }
+}
```
Comment (Lijiaoa) [inline ts/webui/src/static/style/create/topTrial.scss:15]: you could import these scss file from `style/oview/topTrial.scss` directly
other same scss file could also be delete...

File: ts/webui/src/static/style/create/overview.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    //     margin-top: $boxGapPadding;
+
+    //     /* for alert message tooltip position */
+    //     position: relative;
+    // }
+
+    // .duration {
+    //     grid-row: 5 / 9;
+    //     height: 158px;
+    // }
```
Comment (Lijiaoa) [inline ts/webui/src/static/style/create/overview.scss:33]: could be delete

File: ts/webui/src/components/create/table/SuccessTable.tsx [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+    changeExpandRowIDs: Function;
+}
+
+interface SuccessTableState {
+    columns: IColumn[];
+    source: Array<any>;
+    sortInfo: SortInfo;
+}
+
+class SuccessTable extends React.Component<SuccessTableProps, SuccessTableState> {
```
Comment (Lijiaoa) [inline ts/webui/src/components/create/table/SuccessTable.tsx:40]: `file name` should be meaningful, also with `state` and `props` interface name

**Comment by J-shang** [comment]
This pr will be closed, if you are still interested, feel free to contact us, we can continue to improve this feature in another pr.

**Pull request closed without merging**